docs/help-topics/rhtap-import-configure-component/rhtap-import-configure-component.yml

```diff
@@ -11,18 +11,23 @@
 
-    - **Component name**: Is it correct? Click the GitHub link under this field if you want to check your repository URL.
-    - **Runtime**: Did we detect the right one? If so, you should notice that we suggest appropriate deployment options. If you want to change your runtime, expand the menu next to your runtime name and make a selection.
+    - **Component name**: Is it correct? Click the GitHub link under this field if you want to check your repository URL. Save any changes to the name by clicking the checkmark. Discard the changes and exit editing mode by clicking the X.
+    - **Runtime**: Did we detect the right one? If so, you should notice that we suggest suitable deployment options. If you want to change your runtime, expand the menu next to your runtime name and make a selection.
     
     In the **Build & deploy configuration** section, look over these items:
     - **Target port**: Is it correct? If not, click in the field to modify it.
     - **Dockerfile**: If you specified **Dockerfile** as your runtime, make sure we detected the right one. Click in the field if you need to modify it. Don't worry! We hide this field if you chose a different runtime.
-    - **Default build pipeline**: To specify how to trigger rebuilds, toggle to choose either the default build pipeline or a custom one.
-        - **Default build pipeline**: This runs faster because it makes only critical image checks. Consider starting here to make sure we can successfully build and deploy your component.
-        - **Custom build pipeline**: This is triggered when you make commits to your source code repository. This pipeline runs more checks and security scans than the default pipeline, but it takes a bit more time because it's more thorough. **NOTE:** To use a custom pipeline, you must be the owner of your repository so that you can authorize the installation of our application in your GitHub instance and then grant the app access to your repository. If someone else owns your repository, fork it, then go back to the Add components page and start again.
+    - **Default build pipeline**: This switch is on by default. Toggle the switch off to create a **Custom** build pipeline. **Default** and **Custom** build pipelines are triggered differently and include different tasks.
+        - **Default** build pipelines are triggered manually and run faster by skipping some security checks and tests.
+        - **Custom** build pipelines are triggered when you make commits to your source code repository. They run more tests than default pipelines, and you can customize which tasks are included. **NOTE:** To use a custom pipeline, you must be the owner of your repository so that you can authorize our application in your GitHub account and then grant it access to this repository. If someone else owns this repository, fork it, and use the new repo URL when adding the component.
```

Comment:
Would it make sense to mention here that the pipeline type can be changed afterwards (and link to the [upgrade pipeline procedure](https://redhat-appstudio.github.io/docs.appstudio.io/Documentation/main/how-to-guides/configuring-builds/proc_upgrade_build_pipeline/))?